VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionCommentsFragment.java

```diff
@@ -100,7 +116,7 @@ public void onDestroy() {
 
     @SuppressWarnings("unused")
     public void onEventMainThread(DiscussionCommentPostedEvent event) {
-        if (null != event.getParent() && event.getParent().getIdentifier().equalsIgnoreCase(discussionComment.getIdentifier())) {
+        if (null != event.getParent() && event.getParent().getIdentifier().equalsIgnoreCase(discussionResponse.getIdentifier())) {
```

Comment:
Why would we be ignoring the case here? Do we get back inconsistent capitalization? That seems strange